### PR TITLE
Fix conv bn cos_sim cn doc

### PR DESCRIPTION
**Pull request by ceci3**
include the cn doc of:
conv2d
conv3d
conv2d_transpose
conv3d_transpose
batch_norm
cos_sim 
cross_entropy
npair_loss
dropout
img_conv_group

### CODE REVIEW

**Comment by zq19** [comment]
请将PR填写到API整改修复进度表，方便统计进度。

File: doc/fluid/api_cn/layers_cn/conv2d_cn.rst [hunk 1]
```diff
@@ -42,41 +42,37 @@ conv2d
...
+    - **dilation** (int|tuple) - 膨胀比例dilation大小。空洞卷积时会指该参数，滤波器对输入进行卷积时，感受野里每相邻两个特征点之间的空洞信息，根据`可视化效果图<https://github.com/vdumoulin/conv_arithmetic/blob/master/README.md>`_较好理解。如果膨胀比例dialation为元组，则必须包含两个整型数，（dilation_height,dilation_width）。否则，dilation_height = dilation_width = dilation。默认：dilation = 1。
+    - **groups** (int) - 二维卷积层（conv2d layer）的组数。根据Alex Krizhevsky的深度卷积神经网络（CNN）论文中的成组卷积：当group=2，输入和滤波器分别根据通道数量平均分为两组，第一组滤波器和第一组输入进行卷积计算，第二组滤波器和第二组输入进行卷积计算。默认：groups = 1。
+    - **param_attr** (ParamAttr|None) - conv2d 权重的参数属性。可以设置为None或者包含属性的ParamAttr类。如果设为包含属性的ParamAttr类，conv2d创建相应属性的ParamAttr类为param_attr参数。如果param_attr设置为None或者ParamAttr里的初始化函数未设置，参数初始化为Xavier。默认：None。
+    - **bias_attr** (ParamAttr|False|None) - conv2d 偏置参数属性。如果设为False，则卷积不会加上偏置项。如果设为包含属性的ParamAttr类，conv2d创建相应属性的ParamAttr类为bias_attr参数。如果bias_attr设置为None或者ParamAttr里的初始化函数未设置，bias参数初始化为0。默认：None。
+    - **use_cudnn** （bool） - 是否用cudnn核，仅当下载cudnn库才有效。默认：True。
+    - **act** (str) - 激活函数类型，如果设为None，则未添加激活函数。默认：None。
+    - **name** (str|None) - 该层名称（可选）。若设为None，则自动为该层命名。默认：None。
+
+返回：如果未指定激活层，则返回卷积计算的结果，如果指定激活层，则返回卷积和激活计算之后的最终结果。
```
Comment (lanxianghit) [inline doc/fluid/api_cn/layers_cn/conv2d_cn.rst]: 返回没有指明是Tensor还是LoDTensor，请按照规范要求撰写

**Review by lanxianghit — COMMENTED** [review]
文档修改请在pr描述中贴上文档预览图，以方便review

**Comment by ceci3** [comment]
屏幕快照 2019-09-24 下午3 49 09
屏幕快照 2019-09-24 下午2 59 49
屏幕快照 2019-09-24 下午2 59 59
屏幕快照 2019-09-24 下午3 03 37
屏幕快照 2019-09-24 下午3 27 52
屏幕快照 2019-09-24 下午3 28 30
屏幕快照 2019-09-24 下午3 29 46
屏幕快照 2019-09-24 下午3 32 33
屏幕快照 2019-09-24 下午2 58 12
屏幕快照 2019-09-24 下午2 57 31

File: doc/fluid/api_cn/layers_cn/conv2d_cn.rst (hunk 1 above)
Comment (ceci3) [reply doc/fluid/api_cn/layers_cn/conv2d_cn.rst]: done, thanks

File: doc/fluid/api_cn/layers_cn/batch_norm_cn.rst [hunk 2]
```diff
@@ -33,42 +38,42 @@ batch_norm
...
-    - **epsilon** （float，默认1e-05）- 加在分母上为了数值稳定的值。默认值为1e-5。
-    - **param_attr** （ParamAttr|None） - batch_norm参数范围的属性，如果设为None或者是ParamAttr的一个属性，batch_norm创建ParamAttr为param_attr。如果没有设置param_attr的初始化函数，参数初始化为Xavier。默认：None。
-    - **bias_attr** （ParamAttr|None） - batch_norm bias参数的属性，如果设为None或者是ParamAttr的一个属性，batch_norm创建ParamAttr为bias_attr。如果没有设置bias_attr的初始化函数，参数初始化为0。默认：None。
-    - **data_layout** （string,默认NCHW) - NCHW|NHWC。
-    - **in_place** （bool，默认False）- 得出batch norm可复用记忆的输入和输出。
-    - **name** （string，默认None）- 该层名称（可选）。若设为None，则自动为该层命名。
-    - **moving_mean_name** （string，默认None）- moving_mean的名称，存储全局Mean。如果将其设置为None, ``batch_norm`` 将随机命名全局平均值；否则， ``batch_norm`` 将命名全局平均值为 ``moving_mean_name`` 。
-    - **moving_variance_name** （string，默认None）- moving_variance的名称，存储全局变量。如果将其设置为None, ``batch_norm`` 将随机命名全局方差；否则， ``batch_norm`` 将命名全局方差为 ``moving_variance_name`` 。
+    - **input** (Variable) - batch_norm算子的输入特征，是一个Variable类型，输入维度可以是 2, 3, 4, 5。
```
Comment (tink2123) [inline doc/fluid/api_cn/layers_cn/batch_norm_cn.rst]: 补充一下数据类型？

File: doc/fluid/api_cn/layers_cn/batch_norm_cn.rst [hunk 3]
```diff
@@ -33,42 +38,42 @@ batch_norm
...
+    - **bias_attr** （ParamAttr|None） - batch_norm bias参数的属性，可以设置为None或者一个ParamAttr的类（ParamAttr中可以指定参数的各种属性）。如果设为None，默认的参数初始化为0.0。如果在ParamAttr指定了参数的属性时, batch_norm创建相应属性的bias_attr（偏置）参数。默认：None。
+    - **data_layout** （string) - 指定输入数据格式，数据格式可以为NCHW或者NHWC。默认：NCHW。
+    - **in_place** （bool）- batch_norm的输出复用输入的tensor，可以节省显存。默认：False。
+    - **name** （string，默认None）- 该层名称（可选）。若设为None，则自动为该层命名。默认：None。
+    - **moving_mean_name** （string）- moving_mean的名称，存储全局均值。如果将其设置为None, ``batch_norm`` 将随机命名全局均值；否则， ``batch_norm`` 将命名全局均值为 ``moving_mean_name`` 。默认：None。
+    - **moving_variance_name** （string）- moving_variance的名称，存储全局变量。如果将其设置为None, ``batch_norm`` 将随机命名全局方差；否则， ``batch_norm`` 将命名全局方差为 ``moving_variance_name`` 。默认：None。
     - **do_model_average_for_mean_and_var** （bool，默认False）- 是否为mean和variance做模型均值。
-    - **fuse_with_relu** （bool）- 如果为True，batch norm后该操作符执行relu。
-    - **use_global_stats** （bool, Default False） – 是否使用全局均值和方差。 在预测或测试模式下，将use_global_stats设置为true或将is_test设置为true，并且行为是等效的。 在训练模式中，当设置use_global_stats为True时，在训练期间也使用全局均值和方差。
+    - **fuse_with_relu** （bool）- 如果为True，batch_norm后该操作符执行relu。默认：None。
```
Comment (tink2123) [inline doc/fluid/api_cn/layers_cn/batch_norm_cn.rst]: 默认为False？

File: doc/fluid/api_cn/layers_cn/conv2d_cn.rst [hunk 4]
```diff
@@ -42,41 +42,37 @@ conv2d
...
-    - **act** (str) - 激活函数类型，如果设为None，则未添加激活函数。默认：None
-    - **name** (str|None) - 该层名称（可选）。若设为None，则自动为该层命名。
-
-返回：张量，存储卷积和非线性激活结果
-
-返回类型：变量（Variable）
-
-抛出异常:
-  - ``ValueError`` - 如果输入shape和filter_size，stride,padding和group不匹配。
+    - **input** (Variable) - 输入图像，格式为[N,C,H,W]的张量。
```
Comment (tink2123) [inline doc/fluid/api_cn/layers_cn/conv2d_cn.rst]: 补充下数据类型 下同

File: doc/fluid/api_cn/layers_cn/dropout_cn.rst [hunk 5]
```diff
@@ -7,19 +7,17 @@ dropout
...
-    - **seed** (int)-Python整型，用于创建随机种子。如果该参数设为None，则使用随机种子。注：如果给定一个整型种子，始终丢弃相同的输出单元。训练过程中勿用固定不变的种子。
-    - **name** (str|None)-该层名称（可选）。如果设置为None,则自动为该层命名
-    - **dropout_implementation** (string) -
-
-      [‘downgrade_in_infer’(default)|’upscale_in_train’] 其中:
+    - **x** (Variable) - 输入张量。
+    - **dropout_prob** (float32) - 输入单元的丢弃概率，即输入单元设置为0的概率。默认：0.5。
+    - **is_test** (bool) - 标记是否是测试阶段。默认：False。
+    - **seed** (int32) - 整型数据，用于创建随机种子。如果该参数设为None，则使用随机种子。注：如果给定一个整型种子，始终丢弃相同的输出单元。训练过程中勿用固定不变的种子。
+    - **name** (str|None) - 该层名称。如果设置为None，则自动为该层命名。默认：None。
```
Comment (tink2123) [inline doc/fluid/api_cn/layers_cn/dropout_cn.rst]: 使用通用的话术：
- **name** (None|str) – 该参数供开发人员打印调试信息时使用，具体用法请参见 :ref:`api_guide_Name` ，默认值为None。

File: doc/fluid/api_cn/layers_cn/npair_loss_cn.rst [hunk 6]
```diff
@@ -9,31 +9,40 @@ npair_loss
...
 
-NPair损失需要成对的数据。NPair损失分为两部分：第一部分是嵌入向量上的L2正则化器；第二部分是以anchor的相似矩阵和正的相似矩阵为逻辑的交叉熵损失。
+NPair损失需要成对的数据。NPair损失分为两部分：第一部分是对嵌入向量进行L2正则化；第二部分是每一对数据的相似性矩阵的每一行和映射到ont-hot之后的标签的交叉熵损失的和。
 
 参数:
-    - **anchor** (Variable) -  嵌入锚定图像的向量。尺寸=[batch_size, embedding_dims]
-    - **positive** (Variable) -  嵌入正图像的向量。尺寸=[batch_size, embedding_dims]
-    - **labels** (Variable) - 1维张量，尺寸=[batch_size]
-    - **l2_reg** (float32) - 嵌入向量的L2正则化项，默认值：0.002
+    - **anchor** (Variable) -  锚点图像的嵌入向量。尺寸=[batch_size, embedding_dims]。
```
Comment (tink2123) [inline doc/fluid/api_cn/layers_cn/npair_loss_cn.rst]: 数据类型

**Review by kuke — APPROVED** [review]
LGTM
